lang/deutsche.yml

```diff
@@ -0,0 +1,19 @@
+next_move: Was ist Ihr nächster Schritt
```

Comment:
I think a better translation for "Please make your next move" is "Bitte machen Sie Ihren nächsten Zug". 
The word "Schritt" means more something like "step"